### PR TITLE
changed navbar in mobile view

### PR DESCRIPTION
**Pull request by aswinzz**
This PR is for the issue #132

aswinzz.github.io/opencodecollab/

### CODE REVIEW

File: _includes/navbar.html [hunk 1]
```diff
@@ -1,3 +1,148 @@
+<style type="text/css">
+ /* SideNav.css by Mohamed Abo El-Ghranek */
```
Comment (gauthamzz) [inline _includes/navbar.html]: remove this line

File: _includes/navbar.html [hunk 2]
```diff
@@ -11,7 +156,92 @@
...
 
 <!-- Navigation Bar -->
-    <nav class="navbar navbar-default navbar-fixed-top">
+ 
+
+
+
+
+<!-- 
+- SideNav.css by Mohamed Abo El-Ghranek - http://midoghranek.github.io/sidenav
```
Comment (gauthamzz) [inline _includes/navbar.html]: this also remove

File: _includes/navbar.html [hunk 3]
```diff
@@ -11,7 +156,92 @@
 </div>
 
 <!-- Navigation Bar -->
-    <nav class="navbar navbar-default navbar-fixed-top">
+
```
Comment (gauthamzz) [inline _includes/navbar.html]: remove extra unwanted spaces

File: _includes/navbar.html [hunk 4]
```diff
@@ -11,7 +156,92 @@
...
+- SideNav.css by Mohamed Abo El-Ghranek - http://midoghranek.github.io/sidenav
+- To chnage Icons 
+  * follow this link http://fortawesome.github.io/Font-Awesome/icons/
+  * find ths icon code like <i class="fa fa-home"></i>
+  * relace your icon in the HTML code below
+- HTML code example 
+<div id='snav' class='en'><ul>
+  <li><a href='#'><i class="fa fa-home"></i><span>Home</span></a></li> 
+</ul></div>
+-->
```
Comment (gauthamzz) [inline _includes/navbar.html]: also add comments on which part of code refers to which nav bar

File: _data/scores.yml [hunk 5]
```diff
@@ -279,7 +279,7 @@
       score: 30
     - name: Add more solutions to Codechef problems(10 questions)
       link: https://github.com/faheemzunjani/Cool-Practice-Programs/pull/12
-      score: 20
+      score: 40
```
Comment (gauthamzz) [inline _data/scores.yml]: make this 0 . this change will be for everyone

File: _includes/navbar.html [hunk 6]
```diff
@@ -46,7 +276,7 @@
         </div>
       </div>
     </nav>
-<!-- End of Navigation Bar -->
```
Comment (gauthamzz) [inline _includes/navbar.html]: please don't remove comments , they are needed

**Comment by aswinzz** [comment]
done

File: _data/scores.yml (hunk 5 above)
Comment (aswinzz) [inline _data/scores.yml]: 0?

File: _includes/navbar.html [hunk 7]
```diff
@@ -156,48 +136,27 @@
...
-    
     <li>
       <a href="index.html#mentors">
         <i class="fa fa-user-circle-o"></i>
         <span>Mentors</span>
       </a>
     </li>
-    
-     <li>
+         <li>
```
Comment (gauthamzz) [inline _includes/navbar.html]: fix indendation

File: _includes/navbar.html [hunk 8]
```diff
@@ -156,48 +136,27 @@
...
       <a href="index.html#students">
         <i class="fa fa-address-book"></i>
         <span>Participants</span>
       </a>
     </li>
- 
-     <li>
+      <li>
```
Comment (gauthamzz) [inline _includes/navbar.html]: indentation is wrong

File: _includes/navbar.html [hunk 9]
```diff
@@ -227,16 +186,7 @@
...
-     <!-- <li>
-      <a>
-       <div class="fb-like" data-href="https://www.facebook.com/Opencode-1038106262984513/" data-layout="button_count" data-action="like" data-size="large" data-show-faces="false" data-share="true"></div>
-      </a>
-    </li>
-    <li>-->
-    
-     
-
+   </li>
```
Comment (gauthamzz) [inline _includes/navbar.html]: indentation wrong

**Comment by gauthamzz** [comment]
add necessary comments everywhere

**Comment by gauthamzz** [comment]
LGTM :+1: This was a great change @aswinzz

**Pull request merged**